chapters/functions.tex

```diff
@@ -2046,10 +2046,9 @@ \subsubsection{Simple Types}\label{simple-types}
 \end{tabular}
 \end{center}
 
-Sending string literals to FORTRAN~77 subroutines/functions is supported
-for Lapack/Blas-routines, and the strings are \textsc{nul}-terminated for
-compatibility with C. Returning strings from FORTRAN~77
-subroutines/functions is currently not supported.
+Sending string literals to FORTRAN~77 subroutines/functions is supported for Lapack/Blas-routines, and the strings are \textsc{nul}-terminated for compatibility with C.
+The strings should be UTF-8 encoded similarly as for C to be compatible with C, even though the support in FORTRAN~77 is unclear and it is not relevant for the Lapack/Blas-routines.
```

Comment:
I don't see a need to aim for compatibility with C here.  Why not simply this?
```suggestion
Strings are UTF-8 encoded, even though the support for non-ASCII characters in FORTRAN~77 is unclear and it is not relevant for the Lapack/Blas-routines.
```

Reply:
Done.

Reply:
Now it is done.

Reply:
Looks good.  Resolving.